src/main/java/lotto/domain/AnswerSheet.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class AnswerSheet {
+
+    private final List<Integer> numbers;
+    private final Integer bonusNumber;
+
+    public AnswerSheet(List<Integer> numbers, String bonusNumber) {
+       this(numbers,Integer.parseInt(bonusNumber));
+    }
+    public AnswerSheet(List<Integer> numbers, Integer bonusNumber) {
+        if(numbers.size() != 6)
```

Comment:
로또에서 로또의 갯수는 6개라는 매직 넘버는 상수로 분리가 가능하지 않을까요?
```suggestion
        if(numbers.size() != Lotto.LOTTO_NUMBER_LIMIT)
```

Reply:
상수로 분리하였습니다~